### PR TITLE
[TRANSLATION] Common.yml: General keys

### PR DESCRIPTION
**Pull request by thiagoloal**

### CODE REVIEW

**Review by the-physicist — CHANGES_REQUESTED** [review]
Hello, **Thiago**! Thanks for helping us.

I found some typos and standardization errors, that's why I'm requesting changes.
Allow me to make a few points:

The lines below have a typo or spelling error,

```
12 "Nos"; should be "Nós"
16 "acessao"; should be "acesso"
```
and maybe there's some more.

I recommend using more recurring verb tenses than the most 'correct':

```
14 Avoid duplicate responses
Evitar respostas duplicadas ❌
Evite respostas duplicadas ✔️

16 Give you access to your data
Te dar acesso aos seus dados ❌
Lhe dá acesso aos seus dados ✔️
Te dá acesso aos seus dados ✔️
```

Keep the same formatting as the original to standardize:

```
Review Survey 
Revisar pesquisa ❌
Revisar Pesquisa ✔️
```

Hope these suggestions are relevant! If you need anything else, just call me.

File: common.yml [hunk 1]
```diff
@@ -6,187 +6,186 @@ translations:
 
     # homepage
     - key: general.why_create_account
-      t: Why do I need to create an account?
+      t: Por que eu preciso criar uma conta?
     - key: general.create_account_reasons
       t: >
-          We ask you to create an account in order to:
+          Nos pedimos para você criar uma conta para:
```
Comment (thiagoloal) [inline common.yml:12]:
```suggestion
          Nós pedimos para você criar uma conta para:
```

File: common.yml [hunk 2]
```diff
@@ -6,187 +6,186 @@ translations:
...
     - key: general.why_create_account
-      t: Why do I need to create an account?
+      t: Por que eu preciso criar uma conta?
     - key: general.create_account_reasons
       t: >
-          We ask you to create an account in order to:
+          Nós pedimos para você criar uma conta para:
 
-          - Avoid duplicate responses
+          - Evite respostas duplicadas
```
Comment (thiagoloal) [inline common.yml:14]:
```suggestion
          - Evite respostas duplicadas
```
```suggestion
          - Evitar respostas duplicadas
```

**Comment by the-physicist** [comment]
Well done, Thiago! Let's follow that way then.

**Pull request merged**